client/web/src/setup-wizard/components/SyncRepositoriesStep.tsx

```diff
@@ -7,10 +7,12 @@ import { SiteAdminRepositoriesContainer } from '../../site-admin/SiteAdminReposi
 
 import { CustomNextButton } from './setup-steps'
 
-interface SyncRepositoriesStep extends TelemetryProps {}
+interface SyncRepositoriesStep extends TelemetryProps {
+    baseURL: string
+}
 
 export function SyncRepositoriesStep(props: SyncRepositoriesStep): ReactElement {
-    const { telemetryService, ...attributes } = props
+    const { telemetryService, baseURL, ...attributes } = props
```

Comment:
I don't like this style of declaring components and destructuring props, it feels like a source of potential typing mistakes and creates an unnecessary extra `props` variable that we don't use again.

I recommend putting the destructuring in the declaration, declaring the component with a specific type, and deleting the line below (GitHub won't let me suggest that 🤷‍♀️)

```suggestion
export const SyncRepositoriesStep: FC<SyncRepositoriesStep> = ({ telemetryService, baseURL, ...attributes }) => {
```